### ISSUE TITLE
Gamescope

### ISSUE THREAD

**Issue by road2react** [flathub/org.polymc.PolyMC#16]
Is it possible to use gamescope?

**Comment by dada513** [comment]
Probably. I guess I need to add-extension https://github.com/flathub/com.valvesoftware.Steam.Utility.gamescope in the flatpak itself

**Comment by dada513** [comment]
Gamescope is broken on NVIDIA so I can't test it properly though

**Comment by binex-dsk** [comment]
gamescope more like games cope

**Comment by dada513** [comment]
Fixed in v1.2.0: just `flatpak install com.valvesoftware.Steam.utility.gamescope`

**Comment by TacoCake** [comment]
For me, gamescope crashes when trying to use it:

```
/app/utils/gamescope/bin/gamescope: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.30' not found (required by /app/utils/gamescope/bin/gamescope)
/app/utils/gamescope/bin/gamescope: /usr/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /app/utils/gamescope/bin/gamescope)
/app/utils/gamescope/bin/gamescope: /usr/lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.35' not found (required by /app/utils/gamescope/lib/libinput.so.10)`
```

I've found this comment on a similar issue with flatpak gamescope:

> Ah I see. Bottles imports the Steam Utility Plugins via its manifest. Yeah, this will not work when both are on different runtimes.
[source](https://github.com/flathub/com.valvesoftware.Steam.Utility.gamescope/issues/58#issuecomment-1237154874)
